src/main/java/focusedCrawler/crawler/async/HttpDownloader.java

```diff
@@ -69,8 +69,12 @@
     private Counter counterAborted;
     private Counter counterSuccess;
     private Counter counterHttpStatus2xx;
+    private Counter counterHttpStatus401;
+	private Counter counterHttpStatus403;
```

Comment:
Remind to use 4 spaces instead of tabs. You can configure the IDE to use the project's style guide (https://github.com/ViDA-NYU/ache/blob/master/eclipse-code-style.xml) if you haven't done it yet. It is basically the Google style guide but using 4 spaces for indentation.